lib/Connection.js

```diff
@@ -192,6 +192,9 @@ Connection.prototype.connect = function() {
   parser.on('tagged', function(info) {
     self._resTagged(info);
   });
+  parser.on('error', function(error) {
+    self._curReq.bodyEmitter.emit('error', error);
+  });
```

Comment:
new event that must be handled to avoid exception from bubbling up. (will require LIS change)